.tools/runtimes/python.Makefile

```diff
@@ -1,9 +1,16 @@
+.PHONY: install lint format check
+
+# shell
+SHELL := /bin/bash
+
 # python env handler
 VENV := venv/bin
 UV := $(VENV)/uv pip install -p $(VENV)/python
 
 # Development targets
-env:
+env: venv/bin/activate
+
+venv/bin/activate:
```

Comment:
Since we have VENV as a var and the user can overwrrite it (due to `:=`), this should use the variable.

(My bad I only spotted that now - I also didn't have it in original example)

Reply:
Addressed :)